keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -983,6 +983,29 @@ class SqlDeliveryConfigRepository(
     }
   }
 
+  override fun triggerRecheck(application: String) {
+    val uid = sqlRetry.withRetry(READ) {
+      jooq.select(DELIVERY_CONFIG.UID)
+        .from(DELIVERY_CONFIG)
+        .where(DELIVERY_CONFIG.APPLICATION.eq(application))
+        .fetchOne(DELIVERY_CONFIG.UID)
+    }
+
+    if (uid == null) {
+      log.error("Config for app $application does not exist, cannot recheck it")
+      return
+    }
+
```

Comment:
@robfletcher in order to actually make this env immediately recheck I need to clear the leased at field. But, that means that if another instance is checking the env there's a potential to have some cross-contamination. What do you think the best way is to get the environment to be rechecked as soon as possible?

Reply:
Yeah, I think this is legit, although I wonder if it makes sense to clear the `LEASED_AT` field as well? It seems like this will leave it (however briefly) in a weird state where it was leased at time x, but we don't know who by.

Reply:
Okay, I'll clear that field as well for consistency.

Reply:
Isn't setting it to `EPOCH` the equivalent of clearing it, though, since that is the default value? I don't think that column is nullable (or maybe I'm confusing with `AT`?).